Snowflake.Data.Tests/SFBindTestIT.cs

```diff
@@ -56,7 +56,51 @@ public void testArrayBind()
         [Test]
         public void testBindNullValue()
         {
+            using (SnowflakeDbConnection dbConnection = new SnowflakeDbConnection())
+            {
+                dbConnection.ConnectionString = connectionString;
+                dbConnection.Open();
+                try
+                {
+                    using (IDbCommand command = dbConnection.CreateCommand())
+                    {
+                        command.CommandText = "create or replace table TEST_TBL (ID number);";
+                        command.ExecuteNonQuery();
+                    }
+                    
+                    using (IDbCommand command = dbConnection.CreateCommand())
+                    {
+                        command.CommandText = "insert into TEST_TBL values(:p0)";

```

Comment:
So we already support name based binding?

Reply:
Some how this works so yes.